.editorconfig

```diff
@@ -0,0 +1,19 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing-whitespace = true
+insert_final_newline = true
+
+[*.{glsl,rs,toml}]
+indent_style = space
+indent_size = 4
+
+[*.yml]
+indent_style = space
+indent_size = 2
+
+[Makefile]
+indent_style = tab
+indent_size = 4
```

Comment:
Actually, I see no reason why we should add the `indent_size` at all if it's about tabs. With spaces it makes sense but let people decide on their own for something that uses tabs anyways?

Reply:
it'll render 4 spaces for tab on github UI instead of 8 though.

Reply:
I mean that's something anyone who actually cares can override too. I don't particularly care about what github does, I don't write my code for github.

Reply:
yeah, that's fair.